### ISSUE TITLE
Can mdCNN support a convolution3dLayer-function, which Matlab only has for 2d...

### ISSUE THREAD

**Issue by madssakre** [hagaygarty/mdCNN#3]
is it possible to run the mdCNN 3d convolution kernel direction from the standard matlab framework, where layers are defined, e.g.:

layers = [ ...
    imageInputLayer([64,64,6])
   " convolution3dLayer([4,4,6]) "   <---- which would be the mdCNN layer 
...
    fullyConnectedLayer(4096)
...
    regressionLayer];

**Comment by hagaygarty** [comment]
No, the mdCNN layers are not compatible with the standard matlab CNN layers
